service/test/controllers/config/ApplicationConfigControllerTest.java

```diff
@@ -64,10 +66,10 @@ public void testupdateSystemSettings() {
 
       JsonNode json = Json.parse(data);
       RequestBuilder req =
-          new RequestBuilder().bodyJson(json).uri("/v1/system/settings").method("POST");
+          new RequestBuilder().bodyJson(json).uri("/v1/system/settings/set").method("POST");
       /*req.headers(headerMap);*/
       Result result = Helpers.route(application, req);
-      assertEquals(200, result.status());
+      assertEquals(400, result.status());
```

Comment:
Ignore this as of now

Reply:
done

Reply:
I think instead of ignoring this , just fix method nane "testupdateSystemSettings" , name indicated it's success test , but we are expecting 400 , data error. so update method name itself.